### PR TITLE
Milestone 1: Portfolio: setup and mobile version

### PR DESCRIPTION
**Pull request by WilfriedM01**
**Here are the features implemented:**

- Setting up a new repo
- Setting up Html and CSS linter
- Added descriptive README
- Implemented navbar with HTML and CSS only
- Applied correct GitHub Flow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      </header>
+      <section class="bio-content">
+        <div class="bio-content-wrapper">
+          <h1>I’m Wilfried. <br />Glad to see you!</h1>
+          <div class="bio">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </div>
```
Comment (taldr27) [inline index.html]: - [x] Awesome job on your project! **I have a friendly suggestion to improve the HTML structure**. Would you kindly consider changing this div to a p tag? 😁

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello @WilfriedM01, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional an customized Readme. 
✅ Linter tests passing.
✅ Correct use of GitHub Flow.
✅ Nice mobile design.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

File: index.html (hunk 1 above)
Comment (WilfriedM01) [reply index.html]: Good idea :+1: 
Thank you, I apply the change

**Review by ademibowale — APPROVED** [review]
Hi @WilfriedM01  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- The section has the proper background image ✔️
- Descriptive commit messages ✔️ 
- The content is aligned with flexbox ✔️
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

- [ ] [Optional] I will make this suggestion optional as the previous reviewer forgot to mention this to you before, kindly fix your LicenseMit in your Readme, I can see that you don't have a LicenseMit in your directory, kindly create a License file in your directory to fix the License link and make sure the license link is not broken, kindly fix it for a professional reason The MIT license gives express permission for users to reuse code for any purpose, sometimes even if code is part of proprietary software.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**